@ecomplus/storefront-components/src/js/TheProduct.js

```diff
@@ -152,7 +152,7 @@ export default {
       selectedVariationId: null,
       currentGalleyImg: 1,
       isOnCart: false,
-      qntToBuy: 1,
+      qntToBuy: this.product.min_quantity || 1,
```

Comment:
`this.product` pode ser undefined nesse momento porque a prop não é requerida (se fosse teria um `require: true` https://vuejs.org/guide/components/props.html#prop-validation), e nesse caso teria um erro fatal aqui

Reply:
ok